tests/application_development/code_relocation/Kconfig

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
CI failure: missing copyright here.

Reply:
jsut update. Thanks for checking. @povergoing